Client/Frontend/Browser/SessionData.swift

```diff
@@ -6,6 +6,33 @@ import Foundation
 
 import Shared
 
+private func migrate(urls: [URL]) -> [URL] {
+    return urls.compactMap { url in
+        var url = url
+        [("http://localhost:6571/errors/error.html?url=", "\(InternalURL.baseUrl)/\(SessionRestoreHandler.path)?url=")
```

Comment:
Maybe I'm just noticing this, but this doesn't look right. Was the old "session restore" URL really "errors/error.html"?!

Reply:
It was. I have a theory as to why. Because the error page mechanism was in place and then re-used to do session restore handling, and that grew to a much larger body of code.